.mkchlog.yml

```diff
@@ -0,0 +1,23 @@
+# mkchlog was introduced after this commit
+skip-commits-up-to: 1eb6e0c7d85fbc5e3f265de069b91be2cb722359
+
+sections:
+    security:
+        title: Security
+        description: This section contains very important security-related changes.
+        subsections:
+            vuln_fixes:
+                title: Fixed vulnerabilities
+    breaking:
+        title: Breaking changes
+    feat:
+        title: New features
+    bug:
+        title: Fixed bugs
```

Comment:
Is it expected that a 'vuln_fixes' gets a 'bug' section as well?

Reply:
Nothing is ever duplicated. Each record goes into its own single section. While it's true that vulnerability implies bug, duplication is not useful for users I believe.

Reply:
Cool, got it. I agree. Thanks